indico/modules/events/notifications_test.py

```diff
@@ -0,0 +1,46 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2021 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from pathlib import Path
+
+import pytest
+
+from indico.web.flask.templating import get_template_module
+
+
+@pytest.mark.parametrize(('num_events', 'comment', 'snapshot_name'), (
+    (1, '', 'move-request-new-single-nocomment.txt'),
+    (2, '', 'move-request-new-multi-nocomment.txt'),
+    (1, 'Please move this.\nThanks!', 'move-request-new-single-comment.txt'),
+    (2, 'Please move this.\nThanks!', 'move-request-new-multi-comment.txt'),
+))
+def test_move_request_creation_email_plaintext(snapshot, create_category, create_event, num_events, comment,
+                                               snapshot_name):
+    cat = create_category(title='target')
+    events = [create_event(title=f'test {n + 1}') for n in range(num_events)]
+    template = get_template_module('events/emails/move_request_creation.txt',
+                                   events=events, target_category=cat, comment=comment)
+    snapshot.snapshot_dir = Path(__file__).parent / 'templates/emails/tests'
+    snapshot.assert_match(template.get_body(), snapshot_name)
+
+
+@pytest.mark.parametrize(('accept', 'num_events', 'reason', 'snapshot_name'), (
+    (False, 1, '', 'move-request-rejected-single-noreason.txt'),
+    (False, 2, '', 'move-request-rejected-multi-noreason.txt'),
+    (False, 1, 'We do not want your event.\nSorry not sorry.', 'move-request-rejected-single-reason.txt'),
+    (False, 2, 'We do not want your event.\nSorry not sorry.', 'move-request-rejected-multi-reason.txt'),
+    (True, 1, '', 'move-request-accepted-single.txt'),
+    (True, 2, '', 'move-request-accepted-multi.txt'),
+))
+def test_move_request_closed_email_plaintext(snapshot, create_category, create_event, accept, num_events, reason,
+                                             snapshot_name):
+    cat = create_category(title='target')
+    events = [create_event(title=f'test {n + 1}') for n in range(num_events)]
+    template = get_template_module('events/emails/move_request_closure.txt',
+                                   events=events, target_category=cat, accept=accept, reason=reason)
```

Comment:
I'm already including the target category here since I think it'd make sense to mention it in this particular email as well...